### PR TITLE
BigQuery Stream Documentation Updates

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Updated user permissions and re-organized some content to improve readability.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by sanpj2292 — APPROVED** [review]
LGTM

File: destinations/streaming-platforms/bigquery-stream.mdx [hunk 1]
```diff
@@ -47,44 +43,40 @@ To configure BigQuery Stream as a destination in RudderStack, you will need to a
 
 - **Project ID**: Enter your BigQuery project ID here.
 
-- **Dataset ID**: This is the ID of the project dataset associated for which you specified the  **Project ID** above.
+- **Dataset ID**: This is the ID of the project dataset for which you specified the  **Project ID** above.
```
Comment (saikumarrs) [inline destinations/streaming-platforms/bigquery-stream.mdx:46]:
```suggestion
- **Dataset ID**: This is the ID of the project dataset for which you specified the **Project ID** above.
```

File: destinations/streaming-platforms/bigquery-stream.mdx [hunk 2]
```diff
@@ -93,21 +85,23 @@ Consider the following table schema:
...
 <div class="warningBlock">
 
-BigQuery Stream supports only <code class="inline-code">track</code> type of events. Also, make sure the <code class="inline-code">track</code> event payload format matches the schema of the table specified in the RudderStack dashboard.
+ Make sure the <code class="inline-code">track</code> event payload format matches the table schema specified in the RudderStack dashboard.
 </div>
 
 <div class="infoBlock">
 
-RudderStack currently does not support the <code class="inline-code">templateSupportSuffix</code> feature which creates a table schema during a streaming insert action.
+  RudderStack currently does not support the <code class="inline-code">templateSupportSuffix</code> feature which creates a table schema during a streaming insert action.
```
Comment (saikumarrs) [inline destinations/streaming-platforms/bigquery-stream.mdx:101]:
```suggestion
RudderStack currently does not support the <code class="inline-code">templateSupportSuffix</code> feature which creates a table schema during a streaming insert action.
```

File: destinations/streaming-platforms/bigquery-stream.mdx [hunk 3]
```diff
@@ -121,54 +115,54 @@ rudderanalytics.track("event", {
...
+
+4. In the **Select a role** field, search and select the **BigQuery User** role and click on **CONTINUE**.
+
+<img src="../../assets/event-stream-destinations/bqstream-service-account-4.png" alt="BigQuery User role" />
 
-- Then, set the required permissions. Refer to [**this guide**](https://cloud.google.com/bigquery/streaming-data-into-bigquery#required_permissions) for more information on the required IAM permissions.
+5. Click on **DONE** to finish the setup.
 
-- After creating the service account, the service account JSON is needed for RudderStack to send the data to BigQuery. To obtain the JSON, go to your service account and click on it.
+6. Next, you need the service account JSON required for RudderStack to send the data to BigQuery. To obtain this JSON, go to your service account.
```
Comment (saikumarrs) [inline destinations/streaming-platforms/bigquery-stream.mdx:140]: Just to remain consistent with the previous usage.
```suggestion
6. Next, you need the service account credentials JSON required for RudderStack to send the data to BigQuery. To obtain this JSON, go to your service account.
```

**Review by saikumarrs — APPROVED** [review]
LGTM except for the previously added non-blocking comments.

**Review by ishachopra0401 — APPROVED** [review]

**Pull request merged**